Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -485,6 +486,9 @@ LocalFrameView* AsyncScrollingCoordinator::frameViewForScrollingNode(ScrollingNo
         return nullptr;
     
     RefPtr localMainFrame = dynamicDowncast<LocalFrame>(page()->mainFrame());
+    if (!localMainFrame && is<RemoteFrame>(page()->mainFrame()))
+        localMainFrame = dynamicDowncast<LocalFrame>(page()->mainFrame().tree().firstChild());
```

Comment:
This is not correct.  It is not the main frame, and there is nothing special about the first child frame.  It looks like we need to iterate the Page's root frames instead.

Reply:
Updated this to iterate through `page()->rootFrames()`.